src/NosCore.GameObject/Character.cs

```diff
@@ -739,13 +742,13 @@ private NInvPacket BuyFrom(ShopItem item, short amount, short slotChar)
             });
             var sellAmount = (item?.Price ?? 0) * amount;
             Gold += sellAmount;
-            SendPacket(this.GenerateGold());
+            await SendPacket(this.GenerateGold());
             Shop.Sell += sellAmount;
 
-            SendPacket(new SellListPacket
+            await SendPacket(new SellListPacket
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)